analytics/src/main/java/com/mercadopago/sdk/android/analytics/domain/interactor/Analytics.kt

```diff
@@ -0,0 +1,43 @@
+package com.mercadopago.sdk.android.analytics.domain.interactor
+
+import com.mercadopago.sdk.android.analytics.domain.exception.AnalyticsInitializationException
+
+/** Core analytics implementation for the MercadoPago SDK.
+ *
+ * This class is responsible for:
+ * - Collecting events and screen views
+ * - Aggregating environment data
+ * - Formatting and sending analytics data
+ *
+ *  This class have to be initialized first with [initialize] method
+ *  then get a instance by [getInstance]
+ * @param sessionId session identification of this analytics instance
+ * @param siteId site Id of the seller
+ * @param version this application SDK version
+ * */
+class Analytics constructor(
```

Comment:
can we make it internal to avoid usages outside the initialize defined below?

```suggestion
class Analytics internal constructor(
```